packages/e2e-test-utils/src/uploadPublisherLogo.js

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { join, resolve } from 'path';
+import { tmpdir } from 'os';
+import { copyFileSync } from 'fs';
+
+const getFileName = (name) => name.split('.').slice(0, -1).join('.');
```

Comment:
This seems to a copy and paste from https://github.com/google/web-stories-wp/blob/main/packages/e2e-test-utils/src/uploadFile.js#L24

Break this off into another file maybe?